### ISSUE TITLE
createModel does not have definitions for selectors

### ISSUE THREAD

**Issue by BenevidesLecontes** [rematch/rematch#818]
**Describe the bug**
When using createModel the definition type does not have selectors or Im missing something?

- Version 2.0.0-next.5

**Comment by tianzhich** [comment]
We are refactoring the type definitions of the codebases and `@rematch/select` have not done yet. You can track on this: #816

**Comment by BenevidesLecontes** [comment]
@tianzhich ok thanks, for now i'm removing my selectors from my models.

**Comment by tianzhich** [comment]
@BenevidesLecontes Hi, we are looking for contributors that can help us with the refactoring of `select`. PR welcome if you are interested. #816

**Comment by semoal** [comment]
🥳  We released `@rematch@2.0.0-next.7`  🥳  

Feel free to give it a try: 
```sh
npx update-by-scope -t next @rematch
```

### Bug fixes:

 - Typescript compatibility with @rematch/select plugin (we have still work to do it here, but at least it's usable)
- Support optional payload parameter on reducers also on typescript :) 

### Changes to be made?: 

If you were using 

```ts
export type RootState = RematchRootState<RootModel & FullModel>
```

Change it to two generics:

```ts
export type RootState = RematchRootState<RootModel, FullModel>
```

With this change we're closer of a 100% compatibility with typescript eco-system and redux 💯
